src/librustc_target/spec/windows_base.rs

```diff
@@ -87,7 +86,8 @@ pub fn opts() -> TargetOptions {
         ],
         late_link_args,
         post_link_objects: vec![
-            "rsend.o".to_string()
+            "rsend.o".to_string(),
+            "libgcc.a".to_string(), // libgcc or compiler-rt
```

Comment:
This seems like a bit of a worrisome change to me. It probably shouldn't happen on the pc-windows-gnu targets, and only if necessary happen on the uwp targets. (I commented about this above as well)

Reply:
If we always use llvm to build rust for UWP (for the gnu flavor at least) then that should be achievable (provided the folder containing "libclang_rt.builtins-<arch>.a" is in the compiler's path, but that is likely)
If not, then we need to invoke `<compiler> -print-libgcc-file-name` to get the library, and we can't put it in the spec file